pages/examples/web3_authentication.tsx

```diff
@@ -0,0 +1,224 @@
+import * as React from 'react';
+import * as Utilities from '@common/utilities';
+import * as Queries from '@common/queries';
+
+import ActionItem from '@system/documents/ActionItem';
```

Comment:
We need to alphabetize imports, and organize imports based on their shape, can you do this when you get the chance?